### PR TITLE
Clear manifest

### PR DESCRIPTION
**Pull request by steveburton4**

### CODE REVIEW

File: ci/uc-data-load/meta-uc-data-load.yml [hunk 1]
```diff
@@ -36,3 +36,49 @@ meta-uc-data-load:
...
+                if [[ -z $(aws s3 ls "${S3_LOCATION}") ]]; then
+                    break
+                fi
+
+                TIME_TAKEN=$((TIME_TAKEN + 1))
+                sleep 1
+            done
+        inputs:
+        - name: .aws
+        - name: meta
```
Comment (dan-hill2802) [inline ci/uc-data-load/meta-uc-data-load.yml:82]: This input isn't used

File: ci/generate-snapshots/jobs/production-generate-snapshots.yml [hunk 2]
```diff
@@ -12,8 +12,6 @@ jobs:
     trigger: false
   - get: aws-sdx
     trigger: false
-  - get: 4am-monday
```
Comment (dan-hill2802) [inline ci/generate-snapshots/jobs/production-generate-snapshots.yml:15]: Why has this been removed? PR didn't mention it

Comment (steveburton4) [reply ci/generate-snapshots/jobs/production-generate-snapshots.yml:15]: This was a temp last weekend to kick off the pipeline over the weekend so removed it at the same time. Should have mentioned on the PR.

File: ci/uc-data-load/meta-uc-data-load.yml (hunk 1 above)
Comment (steveburton4) [reply ci/uc-data-load/meta-uc-data-load.yml]: Removed it now

File: ci/uc-data-load/jobs/preprod-uc-data-load.yml [hunk 3]
```diff
@@ -26,6 +26,10 @@ jobs:
     - .: (( inject meta.plan.terraform-output-sdx ))
       params:
         TF_WORKSPACE: 'preprod'
+  - .: (( inject meta-uc-data-load.plan.clear-manifest ))
+    config:
+      params:
+        AWS_ACC: '((aws_account.preprod))'
```
Comment (dan-hill2802) [inline ci/uc-data-load/jobs/preprod-uc-data-load.yml:32]: FYI you don't need inverted commas around the account variables, it works with or without

File: ci/uc-data-load/meta-uc-data-load.yml [hunk 4]
```diff
@@ -36,3 +36,48 @@ meta-uc-data-load:
...
+          - |
+            S3_MANIFEST_BUCKET=$(cat terraform-output-ingest/outputs.json | jq -r '.s3_buckets.value.input_bucket')
+            S3_MANIFEST_PREFIX=$(cat terraform-output-internal-compute/outputs.json | jq -r '.manifest_s3_prefixes.value.import')
+
+            S3_LOCATION="s3://${S3_MANIFEST_BUCKET}/${S3_MANIFEST_PREFIX}/" 
+            TIMEOUT=120
+            
+            aws s3 rm "${S3_LOCATION}" --recursive
+
+            TIME_TAKEN=10
```
Comment (dan-hill2802) [inline ci/uc-data-load/meta-uc-data-load.yml:67]: Why is this 10 and not 0?

Comment (steveburton4) [reply ci/uc-data-load/meta-uc-data-load.yml:67]: No idea, I didn't want to change the logic, so kept it -> it was 10 in the wrapper script.

File: ci/uc-data-load/jobs/preprod-uc-data-load.yml (hunk 3 above)
Comment (steveburton4) [reply ci/uc-data-load/jobs/preprod-uc-data-load.yml:32]: Ah ok, will keep it in this one as it matches the rest of the pipeline, but won't use it for future ones I do.

**Comment by steveburton4** [comment]
@dan-hill2802 if you have the UC mac available, would appreciate the internal compute companion one being looked at please? So I can get it across the pipelines.

**Pull request merged**